go.mod

```diff
@@ -3,26 +3,20 @@ module github.com/open-cluster-management/config-policy-controller
 go 1.16
 
 require (
-	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
-	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
-	github.com/google/go-cmp v0.5.2
-	github.com/onsi/ginkgo v1.14.1
-	github.com/onsi/gomega v1.10.2
+	github.com/ghodss/yaml v1.0.0
+	github.com/golang/glog v1.0.0
+	github.com/google/go-cmp v0.5.5
+	github.com/onsi/ginkgo v1.16.4
```

Comment:
I'm surprised to see ginkgo and gomega as dependencies since they are installed separately as part of the `e2e-dependencies` Make target.

Reply:
Don't most of the controllers have these in the go.mod? I think it's required here since tests import them.